packages/@expo/metro-runtime/src/HMRClient.ts

```diff
@@ -150,7 +150,8 @@ const HMRClient: HMRClientNativeInterface = {
     assert(!hmrClient, 'Cannot initialize hmrClient twice');
 
     const serverScheme = window.location.protocol === 'https:' ? 'wss' : 'ws';
-    const client = new MetroHMRClient(`${serverScheme}://${window.location.host}/hot`);
+    const host = global.metroHost || window.location.host
```

Comment:
where does this get set? Maybe we can pass an arg in instead of using the global?

Reply:
We can't pass arg in packages/@expo/metro-runtime/src/messageSocket.ts code